### PR TITLE
Webpack dev server

### PR DESCRIPTION
**Pull request by gurdasnijor**
This PR makes getting up and running a little faster with the webpack development server.

You'd be able to just `npm run dev` to launch it.

### CODE REVIEW

**Comment by SanderSpies** [comment]
Thanks for this PR - will look into it the coming days.

**Comment by SanderSpies** [comment]
Apologies for the delay - mostly been busy with positive personal stuff.

**Comment by gurdasnijor** [comment]
No worries, looking at it, I think I want to clean this up a bit more, there is some unnecessary stuff in `webpack.config.js`

File: webpack.config.js [hunk 1]
```diff
@@ -3,10 +3,17 @@
...
+var path = require('path');
+
+var port = JSON.parse(process.env.npm_package_config_port || 6000);
+
 
 module.exports = {
 	devtool: 'sourcemap',
-	entry: "./views/DocumentationApplication.js",
+	entry: [
+      'webpack-dev-server/client?http://localhost:' + port,
```
Comment (sokra) [inline webpack.config.js]: better use `--inline` for the webpack-dev-server. It'll add this automatically...

**Pull request closed without merging**